tm2/pkg/std/memfile.go

```diff
@@ -101,6 +103,8 @@ func SplitFilepath(filepath string) (dirpath string, filename string) {
 		return strings.Join(parts[:len(parts)-1], "/"), last
 	} else if last == "" {
 		return strings.Join(parts[:len(parts)-1], "/"), ""
+	} else if last == licenseName {
```

Comment:
Can we please transform this if spaghetti into a switch?

Reply:
Yes, done